test/cluster/dtest/audit_test.py

```diff
@@ -0,0 +1,1408 @@
+
+        yield
+
+        # Remember audit entries after executing the query.
+        rows_after = self.get_audit_log_list(session)
+        set_of_rows_after = set(rows_after)
+        assert len(set_of_rows_after) == len(rows_after), f"audit table contains duplicate rows: {rows_after}"
+
+        new_rows = rows_after[len(rows_before) :]
+        assert set(new_rows) == set_of_rows_after - set_of_rows_before, f"new rows are not the last rows in the audit table: {rows_after}"
+
+        if merge_duplicate_rows:
+            new_rows = self.deduplicate_audit_entries(new_rows)
+
+        assert len(new_rows) == len(expected_entries), f"Expected {len(expected_entries)} new audit entries, but got {len(new_rows)} new entries: {new_rows}"
+
+        for row, entry in zip(new_rows, expected_entries):
+            self.assert_audit_row_eq(row, entry.category, entry.statement, entry.table, entry.ks, entry.user, entry.cl, entry.error)
+
+    def verify_keyspace(self, audit_settings=None, helper=None):
+        """
+        CREATE KEYSPACE, USE KEYSPACE, ALTER KEYSPACE, DROP KEYSPACE statements
+        """
+        session = self.prepare(create_keyspace=False, audit_settings=audit_settings, helper=helper)
+
+        def execute_and_validate_audit_entry(query, category, **kwargs):
+            return self.execute_and_validate_audit_entry(session, query, category, audit_settings, **kwargs)
+
+        execute_and_validate_audit_entry(
+            "CREATE KEYSPACE ks WITH replication = { 'class':'SimpleStrategy', 'replication_factor':1} AND DURABLE_WRITES = true",
+            category="DDL",
+        )
+        execute_and_validate_audit_entry(
+            'USE "ks"',
+            category="DML",
+        )
+        execute_and_validate_audit_entry(
+            "ALTER KEYSPACE ks WITH replication = { 'class' : 'NetworkTopologyStrategy', 'dc1' : 1 } AND DURABLE_WRITES = false",
+            category="DDL",
+        )
+        execute_and_validate_audit_entry(
+            "DROP KEYSPACE ks",
+            category="DDL",
+        )
+
+        # Test that the audit entries are not added if the keyspace is not
+        # specified in the audit_keyspaces setting.
+        keyspaces = audit_settings["audit_keyspaces"].split(",") if "audit_keyspaces" in audit_settings else []
+        assert "ks2" not in keyspaces
+        query_sequence = [
+            "CREATE KEYSPACE ks2 WITH replication = { 'class':'SimpleStrategy', 'replication_factor':1} AND DURABLE_WRITES = true",
+            'USE "ks2"',
+            "ALTER KEYSPACE ks2 WITH replication = { 'class' : 'NetworkTopologyStrategy', 'dc1' : 1 } AND DURABLE_WRITES = false",
+            "DROP KEYSPACE ks2",
+        ]
+
+        with self.assert_no_audit_entries_were_added(session):
+            for query in query_sequence:
+                session.execute(query)
+
+    @pytest.mark.require("scylladb/scylla-enterprise#3236")
+    @pytest.mark.require("scylladb/scylla-enterprise#3590")
```

Comment:
Please, remove `require` markers. We don't need them.

And, btw, @andrzej-jackowski-scylladb suggested to close scylladb/scylla-enterprise#3236 after merge of https://github.com/scylladb/scylla-dtest/pull/5649

scylladb/scylla-enterprise#3590 is merged.